client/src/models/mockJobDataObjects.js

```diff
@@ -0,0 +1,203 @@
+const backendStudent = {
+  header: {
+    position: ["Coop"],
+    title: "Backend Developer",
+    company: "Insert Company Name",
+    location: "Location",
+    startDate: "1999-12-31",
+    length: "8 months",
+  },
+  requirements: {
+    languages: ["Java", "C++", "C"],
+    frameworks: ["Express", "Django", "CakePHP", "MongoDB"],
+    tools: ["GitHub"],
+    concepts: [
+      "Object Oriented Programming",
+      "Asynchronous Programming",
+      "Recursion",
+    ],
+    experience: "≥ 0 months",
+    gpa: "Required",
+    gpaValue: "70",
+  },
+  details: {
+    academicReq: ["Bachelor's Degree"],
+    description: "Insert Description",
+    position: "In-person",
+    pay: "Salary",
+    candidates: "Anyone",
+    coOp: "No",
+  },
+  contact: {
+    name: "Insert Name",
+    email: "Insert Email",
+    linkedIn: "Insert LinkedIn",
+    other: "Insert Other Information",
+    applicationSteps: "Insert Application Steps",
+  },
+};
+
+
+const frontendStudent = {
```

Comment:
change to camel casing: `frontEndStudent`

Reply:
changed